.github/workflows/ci.yml

```diff
@@ -94,14 +94,6 @@ jobs:
           endtoend_suite: asset-admin
           endtoend_config: vendor/silverstripe/asset-admin/behat.yml
           endtoend_tags: job3,job4,job5,job6
-        - php: 8.3
-          endtoend: true
-          endtoend_suite: blog
-          endtoend_config: vendor/silverstripe/blog/behat.yml
```

Comment:
Keep the ones we're making CMS 6 compatible, or else how will we know they're compatible?

Reply:
Updated, and add subsites back in too